README.md

```diff
@@ -47,10 +47,15 @@ is written in C++.
 ## Requirements
 
 Besides the listed requirements (see `requirements.txt`), the random forest
-used in SMAC3 requires SWIG (>= 3.0).
+used in SMAC3 requires SWIG (<4.0, >= 3.0) as a build dependency. After successful installation of `pyrfr`, SWIG can be uninstalled.
```

Comment:
Actually, I would like to avoid to tell people what they can uninstall. The next time they try to install SMAC they might wonder why last time it worked and now it does not

Reply:
The formulation is up to you, but i think its crucial to tell, that it IS and Only IS a build dependency. It took me 3 hours to figure out why it has no effect whatsoever i do with swig unless i actually reinstall pyrfr with that configuration.

Reply:
I agree that we should mention that it is a build dependency. But could you please drop the second sentence.

Reply:
Just hit this from Ubuntu 20.04. Good thing it was documented.